virttest/libvirt_storage.py

```diff
@@ -544,3 +544,23 @@ def clone_volume(self, old_name, new_name):
             logging.info("Volume '%s' does not exist or '%s' has been exist."
                          % (old_name, new_name))
             return False
+
+
+def check_qemu_image_lock_support():
+    """
+    QEMU commit 244a566(qemu-2.10) introduced the image locking feature which brought
+    a new option '-U' to the qemu-img command:info, compare, check, bench, convert,
+    dd, map, snapshot, rebase. This method provides one way to determine whether
+    current qemu-img support or not.
+
+    :return: True if current qemu-img command support
+    """
+    cmd = "qemu-img"
+    try:
+        binary_path = process.system_output("which %s" % cmd)
+    except process.CmdError:
+        raise process.CmdError(cmd, binary_path,
+                               "qemu-img command is not found")
+    cmd_result = process.run("qemu-img" + ' -h', ignore_status=True,
```

Comment:
sometimes it might not be in global path, that's why it is better to use with binary_path.

Reply:
updated
thx